src/test/java/api/support/builders/LoanPolicyBuilder.java

```diff
@@ -34,6 +34,8 @@
   private final boolean loanable;
   private final Period recallsMinimumGuaranteedLoanPeriod;
   private final Period recallsRecallReturnInterval;
+  private final Period alternateRecallReturnInterval;
+  private final boolean allowRecallsToExtendOverudeLoans;
```

Comment:
I think there is a typo in `overdue`.

```suggestion
  private final boolean allowRecallsToExtendOverdueLoans;
```